chore: update changelog (#2154)
berops/claudie: docs/CHANGELOG/changelog-0.15.x.md

# Claudie `v0.15`

!!! warning "Upgrade to this release from previous `v0.14.x` version requires manual intervention (due to the MongoDB version upgrade)."

## Readme Before Deploying 

### Autoscaler changes

> The autoscaler will no longer be deployed by Claudie, and will now be an opt-in feature behind a paywall.

Reviewer: ```suggestion
The autoscaler will no longer be deployed by Claudie, and will now be an opt-in commercial feature.
```
Nitpick but: "feature behind a paywall" sound too blunt to me, people are sensitive for the "paywall" word, it has kind of negative reputation. I would consider changing the wording for something else. @bernardhalas WDYT?

Reviewer: Or `opt-in premium feature`

> The autoscaler will no longer be deployed by Claudie, and will now be an opt-in feature behind a paywall.

Reviewer 2: ```suggestion
The autoscaler will no longer be deployed by Claudie. From now it becomes an opt-in paid feature.
```
